proxy/coco/src/config.rs

```diff
@@ -13,6 +13,10 @@ lazy_static::lazy_static! {
     /// Localhost binding to any available port, i.e. `127.0.0.1:0`.
     pub static ref LOCALHOST_ANY: SocketAddr =
         SocketAddr::V4(SocketAddrV4::new(Ipv4Addr::new(127, 0, 0, 1), 0));
+
+    /// Binds to all local addresses and any available port.
```

Comment:
```suggestion
    /// Binds to all local interfaces and any available port.
```

To keep it consistent with the RFC: https://man7.org/linux/man-pages/man7/ip.7.html

Reply:
Good catch.